src/Transients.cc

```diff
@@ -49,7 +49,7 @@ Transients::init()
     map->cleaner = this;
     map->disableHitValidation(); // Transients lacks slices to validate
 
-    locals = new Locals(entryLimit, 0);
+    locals = new Locals(entryLimit, nullptr);
```

Comment:
This is one of the examples where clang-tidy was able to replace `0` with nullptr.